README.md

```diff
@@ -1,36 +1,78 @@
+## Vocabularies
 
-* archive/data_0.3.April_2014.tar.gz: These data files were in use from April 2014 to September 2014. The database.txt file contains activation data for approximately 8,000 studies. The features.txt files containsfeature data for nearly 3,000 automatically extracted terms. These weights were derived from only the abstract of article. Thus, a value of, say, 0.003 for a feature like "emotion" would indicate that the term 'emotion' constitutes 0.3% of all word tokens in the article abstract. The first (identifying) column is the PubMed ID (PMID) of each article.
+The raw text used to annotate the Neurosynth database is the article abstract,
+which we cannot share directly but which can be downloaded easily from PubMed with a tool like [`nimare.extract.download_abstracts`](https://nimare.readthedocs.io/en/latest/generated/nimare.extract.download_abstracts.html#nimare.extract.download_abstracts).
 
-* archive/data_0.4.September_2014.tar.gz: The database.txt file contains activation data for over 9,700 studies. The features.txt files contains feature data for nearly 3,000 automatically extracted terms, including both single words and N-grams extracted from the Cognitive Atlas vocabulary. These weights were derived from only the abstract of article. In contrast to previous datasets, the feature weights are now normalized, and reflect [tf-idf](http://en.wikipedia.org/wiki/Tf%E2%80%93idf) values rather than proportions. The first (identifying) column is the PubMed ID (PMID) of each article.
+However, the actual annotations are available in this repository.
+Neurosynth currently includes two annotation approaches, resulting in 1-5 vocabularies, depending on the database version.
 
-* archive/data_0.5.February_2015.tar.gz: The format and structure of all files in this release is identical to the previous one (0.4); the only difference is the addition of new studies and features. The 0.5 release contains data for > 10,900 studies and > 3,300 features.
+- `vocab-terms`: This vocabulary refers to terms extracted from abstracts using a vectorizer.
+  Prior to version 0.5, these terms were simply individual words.
+  However, from version 0.5 on, these terms have included both individual words (unigrams) and two-word terms (bigrams).
+  Terms are extracted on a largely unsupervised basis; namely, some words are *not* extracted, based on a "stop list",
+  while others may appear too frequently (e.g., common words like "the") or too infrequently to be meaningful.
+  The current version's terms vocabulary is what is accessed on the Neurosynth website.
+  Additionally, the values for the `features.npz` for this vocabulary are [tf-idf](https://en.wikipedia.org/wiki/Tf–idf) values, starting at version 0.4.
+  In versions prior to 0.4, these values were proportions.
```

Comment:
I just want to make sure this is accurate. If so, I plan to rename the `type-tfidf` files from version 0.3 to `type-proportion`.